src/context/Authenticated.tsx

```diff
@@ -19,13 +19,13 @@ export function AuthenticatedOnlyContext({ children, hideSpinner }: Props) {
             <OnLoadSpinnerProvider defaultState={!hideSpinner}>
                 <QueryParamProvider>
                     <ZustandProvider>
-                        <AppGuards>
-                            <PreFetchDataProvider>
+                        <UserInfoSummaryStoreProvider>
```

Comment:
We want to fetch this **ASAP** so we can use the details in anything else. So this will pretty much always stay super high.